tests/system/src/test/java/test/robot/javafx/scene/RTLTextFlowCharacterIndexTest.java

```diff
@@ -113,7 +113,7 @@ public class RTLTextFlowCharacterIndexTest {
     static volatile Stage stage;
     static volatile Scene scene;
 
-    static final int WIDTH = 500;
+    static final int WIDTH = 600;
```

Comment:
one suggestion: maybe we can reduce the font size from 48 to something smaller.
that might also speed up the test as the cursor wouldn't have to traverse a lot of pixels.  watch out for change in line breaks though.

Reply:
Thanks for suggesting this @andy-goryachev-oracle. I have made the changes in the test. I reduced the font size little bit only to make sure that the multiline text are not displayed in a single line.